src/dev/src/app/datagrid/utils/color-filter.ts

```diff
@@ -10,78 +10,78 @@ import { SerializableFilter } from '../../../../../clr-angular/data/datagrid/int
 
-    constructor() {
-      this._id = Math.random().toString();
-    }
+  constructor() {
+    this._id = Math.random().toString();
+  }
 
-    setId(id: string) {
-      this._id = id;
-    }
+  get id() {
+    return this._id;
+  }
 
-    get id() {
-      return this._id;
-    }
+  public get filterState(): ColorFilterStateInterface {
+    return {
+      id: this.id,
+      type: 'ColorFilter',
+      allColors: this.allColors,
+      selectedColors: this.selectedColors,
+    };
+  }
 
-    public get filterState(): ColorFilterStateInterface {
-      return {
-        id: this.id,
-        type: 'ColorFilter',
-        allColors: this.allColors,
-        selectedColors: this.selectedColors,
-      };
+  public set filterState(state: ColorFilterStateInterface) {
+    this.selectedColors = {};
+    this.nbColors = 0;
+    for (let color in state.selectedColors) {
+      if (state.selectedColors[color]) {
+        this.toggleColor(color);
+      }
     }
+  }
 
-    public set filterState(state: ColorFilterStateInterface) {
-        this.selectedColors = {}
-        this.nbColors = 0;
-        for (let color in state.selectedColors) {
-          if (state.selectedColors[color]) {
-            this.toggleColor(color);
-          }
-        }
-    }
+  setId(id: string) {
+    this._id = id;
+  }
 
-    equals(other: ColorFilter) {
-      return other.filterState.type === this.filterState.type && other.filterState.id === this.filterState.id;
-    }
+  equals(other: ColorFilter): boolean {
+    return other.filterState.type === this.filterState.type && other.filterState.id === this.filterState.id;
+  }
 
-    listSelected(): string[] {
-        const list: string[] = [];
-        for (const color in this.selectedColors) {
-            if (this.selectedColors[color]) {
-                list.push(color);
-            }
-        }
-        return list;
+  listSelected(): string[] {
+    const list: string[] = [];
+    for (const color in this.selectedColors) {
+      if (this.selectedColors[color]) {
+        list.push(color);
+      }
     }
+    return list;
+  }
 
-    toggleColor(color: string) {
-        this.selectedColors[color] = !this.selectedColors[color];
-        this.selectedColors[color] ? this.nbColors++ : this.nbColors--;
-        this.filterState.selectedColors = this.selectedColors;
-        this.changes.emit(true);
-    }
+  toggleColor(color: string): void {
+    this.selectedColors[color] = !this.selectedColors[color];
+    this.selectedColors[color] ? this.nbColors++ : this.nbColors--;
+    this.filterState.selectedColors = this.selectedColors;
+    this.changes.emit(true);
+  }
 
-    accepts(user: User) {
-        return this.nbColors === 0 || this.selectedColors[user.color];
-    }
+  accepts(user: User): boolean {
+    return this.nbColors === 0 || this.selectedColors[user.color];
+  }
 
-    isActive(): boolean {
-        return this.nbColors > 0;
-    }
+  isActive(): boolean {
+    return this.nbColors > 0;
+  }
```

Comment:
This file is just full of indentation and formatting changes which clutters up the diff

Reply:
There is a pre-commit git-hook, that formats the code. But I can disable it, if you would like to.

![image](https://user-images.githubusercontent.com/40567748/52272070-e1d7a880-2945-11e9-9105-b82aca6f3c6d.png)